src/index.js

```diff
@@ -16,20 +17,7 @@ export default function App () {
         </Routes>
       </BrowserRouter>
       <Services />
-      <div className='service-map'></div>
-      <div className='flex flex-col p-16 bg-green-700'>
-        <h1 className='text-center text-white'>
-          Icons from &quot;Lawn Care&quot; collection, created by Berkah Icon,
-          from{' '}
-          <a
-            className='underline'
-            href='https://thenounproject.com/browse/collection-icon/lawn-care-163681/?p=1'
-          >
-            Noun Project
-          </a>{' '}
-          CC BY 3.0
-        </h1>
-      </div>
```

Comment:
Can you please leave the existing footer contents as is since it is out of scope to remove or change it in this story card?